.github/workflows/test_models.yaml

```diff
@@ -139,7 +139,7 @@ jobs:
           fi
           sudo tar -xzf ${CACHED_DOWNLOAD_DIR}/geckodriver-v${GECKO_VER}-linux64.tar.gz -C /usr/local/bin
           geckodriver --version
-          pip install selenium==4.2.0
+          pip install selenium==4.3.0
```

Comment:
I think we can remove these once we are ok with selenium 4.3.0

Reply:
@guynir42 Should we leave this in until https://github.com/cesium-ml/baselayer/pull/330/files get merged maybe?

Reply:
I'm not sure. If the tests pass without this line we should get rid of it so we don't forget to do it later.